tests/unit/style/properties/serialization.rs

```diff
@@ -753,7 +771,7 @@ mod shorthand_serialization {
             let attachment = single_vec_keyword_value!(attachment, scroll);
 
             let image = single_vec_value!(image,
-                Some(Image::Url(SpecifiedUrl::new_for_testing("http://servo/test.png"))));
```

Comment:
I don't think these changes are necessary. For example this line was fine before and we don't like too much empty lines like previous lines. Please revert these.